runtime/doc/ui.txt

```diff
@@ -638,6 +657,8 @@ tabs.
 	When |ui-messages| is active, no message grid is used, and this event
 	will not be sent.
 
+	`zindex` and `compindex` have the same meaning as for `win_float_pos`.
```

Comment:
to avoid confusion could mention that zindex is hardcoded to be `200` but included for completeness, or something.

Reply:
Yes, I will add that when I make the other changes.